### PR TITLE
updated new post modal

### PR DESCRIPTION
**Pull request by 1010varun**
Screenshot from 2023-02-22 15-22-39

@narayan954, I have implemented a new post modal and I would appreciate it if you could review my pull request and merge it.
I understand that you had previously mentioned creating a new branch for new issues, but I have already made these changes and cannot create a new branch at the moment. However, I will make sure to create a new branch for future issues.

Thank you.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -294,8 +294,15 @@ function App() {
           </div>
         )}
       </div>
-      <Dialog open={openNewUpload} onClose={() => setOpenNewUpload(false)}>
-        <DialogTitle>New Upload</DialogTitle>
+      <Dialog style={{borderRadius: "100px"}} open={openNewUpload} onClose={() => setOpenNewUpload(false)}>
```
Comment (narayan954) [inline src/App.jsx:297]: using reserved prop-names like className and style(in this case) on custom components hurts readability and makes the code harder to maintain. These props should only be used for DOM nodes like div, a, span, etc.
You can take a look into this [mui.com/material-ui/customization/how-to-customize](https://mui.com/material-ui/customization/how-to-customize/) and fix it.

File: src/App.jsx [hunk 2]
```diff
@@ -294,8 +294,15 @@ function App() {
...
+        <div style={{padding: "20px", borderRadius: "10%"}}>
+        <center><img
```
Comment (narayan954) [inline src/App.jsx:299]: don't use deprecated tags like centered and update it with modern css alignment approach. You might refer to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center

**Review by narayan954 — CHANGES_REQUESTED** [review]
I have marked some errors, please resolve them. Also, try to style the modal like picture you've provided.
new post modal

**Review by narayan954 — CHANGES_REQUESTED** [review]
still, center tag exists, please fix it

**Comment by 1010varun** [comment]
@narayan954, I have removed all the center tags.

Kindly review and merge my pr.

**Review by narayan954 — CHANGES_REQUESTED** [review]
this workaround of center tag isn't responsive for other screens. You can check the mobile view for reference
mobile-view

using marginleft property won't make the elements centered and responsive here. You can use flexbox/grid to center the elements

**Comment by 1010varun** [comment]
image
 
@narayan954 is this perfect

**Comment by narayan954** [comment]
> image
> 
> image
> 
>  
> @narayan954 is this perfect

No, nothing is perfect, but in this case I'm suggesting you not to disturb the current alignment of other components as it looks in the screenshot below.
[]()

**Comment by 1010varun** [comment]
image


@narayan954 now is this correct

**Comment by 1010varun** [comment]
@narayan954 now please review and merge my pr.

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by narayan954** [comment]
thanks for your contribution

**Pull request merged**